deployments/docker/src/certs/gen-certs

```diff
@@ -0,0 +1,228 @@
+#!/bin/bash
+# Copyright 2024 Contributors to the Veraison project. SPDX-License-Identifier: Apache-2.0
+# SPDX-License-Identifier: Apache-2.0
+set -e
+
+ROOT_CERT_NAME=rootCA
+DEFAULT_DOMAIN=veraison-net
+DEFAULT_SUFFIX=-service
+
+SERVICES=(provisioning verification management keycloak vts)
+THIS_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+function create_root_cert() {
+	_check_openssl
+	pushd "$THIS_DIR" > /dev/null || exit 1
+
+	openssl ecparam -name prime256v1 -genkey -out ${ROOT_CERT_NAME}.key
```

Comment:
I believe you can add a `-noout` here to avoid stashing the (redundant) EC parameters alongside the private key.

Reply:
Thire is no issue in keeping those; and means private key can potentially be used on its own.

Reply:
it can be used without parameters nevertheless because the same information is stashed in the `parameters` field:
```
ECPrivateKey ::= SEQUENCE {
  version        INTEGER { ecPrivkeyVer1(1) } (ecPrivkeyVer1),
  privateKey     OCTET STRING,
  parameters [0] ECParameters {{ NamedCurve }} OPTIONAL,
  publicKey  [1] BIT STRING OPTIONAL
}
```

Anyway, this is not a blocker.  Feel free to ignore this comment.